lib/manager/dockerfile/extract.js

```diff
@@ -1,3 +1,23 @@
+/**
+ * TODO remove theese comments after conversation resolved.
+ * I think it is good Idea to move up file hierarchy all primitives
+ * like strings literals, booleans and numbers.
+ * Store enums in one place (like Android framework demands)
+ * will enable you to make changes easily in the future.
+ * It will be nice to declare [multiStageDockerBuildEnums] (and in the
+ * long term all other strings literals) inside the
+ * enums responsible file of the project and then import
+ * here as [m], then access value in concise way e.g. [m.STAGE].
+ * Just by inspecting enums file developer will be able to quick
+ * understand all possible states of entity without scouring the project.
```

Comment:
On the other hand, if constants/enums are used in only one place in the repo like here, it's nice to have it in one place so a developer doesn't need to go off and find it in a separate file. I usually prefer to centralise definitions only when they are used from multiple places.